@here/harp-examples/src/datasource_here-webtile.ts

```diff
@@ -7,8 +7,46 @@ import { MapControls, MapControlsUI } from "@here/harp-map-controls";
 import { CopyrightElementHandler, MapView } from "@here/harp-mapview";
 import { HereWebTileDataSource, WebTileDataSource } from "@here/harp-webtile-datasource";
 
+import { Theme } from "@here/harp-datasource-protocol";
+import { ProjectionType } from "@here/harp-geoutils";
 import { apikey } from "../config";
 
+const FLAT_THEME: Theme = {
+    clearColor: "#f8fbfd"
```

Comment:
don't you need an `extends` ?

Reply:
no b/c the WebTile source needs no styling

Reply:
Ok, cool.